src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Models/ManageApprenticesFilterModel.cs

```diff
@@ -70,20 +130,58 @@ public class ManageApprenticesFilterModel : ManageApprenticesFilterModelBase
                     {
                         Label = "Next",
                         AriaLabel = "Next page",
-                        RouteData = BuildRouteData(PageNumber + 1)
+                        RouteData = BuildPagedRouteData(PageNumber + 1)
                     });
                 }
 
                 return links;
             }
         }
 
-        private Dictionary<string, string> BuildRouteData(int pageNumber)
+        public Dictionary<string, string> RouteData => BuildRouteData();
+
+        private Dictionary<string, string> BuildRouteData()
         {
-            var routeData = new Dictionary<string, string>
+            var routeData = new Dictionary<string, string>();
+
+            if (!string.IsNullOrWhiteSpace(SearchTerm))
             {
-                {"pageNumber", pageNumber.ToString()}
-            };
+                routeData.Add("searchTerm", SearchTerm);
```

Comment:
is it worth using nameOf() instead of the string?

Reply:
yes